docs/pipeline/portal_run_id.md

```diff
@@ -0,0 +1,95 @@
+# Portal Run ID
+
+You may come across people mentioning this term. The following document and explain about its attributes, property, constraint rule and use cases that intend to solve.
+
+> In summary, the `portal_run_id` is formula-based and, systematic scheme for herding the analysis run records into Portal Pipeline database; either with data "late binding" process or automated tracking process; so that one person (or a system event) would start the analysis without the wait and, still be consistent by following standard operation procedure (SOP) and best practises.
+
+![portal_run_id.png](../model/portal_run_id.png)
+
+link: `docs/model/portal_run_id.png`
+
+## Curation
+
+### Where would you encounter?
+
+Often written as `portal_run_id` in Portal UI, [API endpoints](../ENDPOINTS.md) or data export from Portal database through [Portal Athena](../athena) or via meeting/Slack/Trello communication. This ID keeps track of every BioInformatics analysis run; also known as a "workflow run" for short. You may encounter this ID scheme when someone referring to analysis output for traceability with Portal ecosystem.
+
+### Unit of work
+
+It is designed in mind such that
+- Every BioInformatics analysis run is immutable and single unit of work transaction
+- Every BioInformatics analysis run get assigned one unique `portal_run_id`
+  - This includes re-running the analysis for the same Sample other everytime
+- Each analysis run must meet their terminal state such as `Failed`, `Aborted`, `Succeeded`, `Deleted`
+
+A typical BioInformatics workflow (languages like [CWL](https://www.commonwl.org), [WDL](https://openwdl.org), [Nextflow](https://www.nextflow.io), [Snakemake](https://github.com/snakemake/snakemake)) consist of multiple steps or tasks. A Portal orchestration views one analysis workflow run as single unit of work transaction and, this workflow must either be completed successfully or, failed otherwise.
+
+The complexity of a workflow contains many steps or, small steps within it; is upto the author of BioInformatics workflow expert concern. i.e. A workflow being too big or small. Regardless, Portal orchestration engine takes single workflow run as one unit of work item that must meet terminal condition without tampering the workflow execution run, _in-flight_.
+
+### History
```

Comment:
Minor comment: 
Move/mention the "states" here as they form the history. Make a separate point/emphasis on deletion, it's part of the history, but a common/important use case that's usually user driven (as opposed to automation).

Reply:
Eh. Perhaps we do this editorial together.

Reply:
Sorry Flo. Time is not on our side. Since it is minor edit need, I will go ahead and merge it as-is for now. We can come back to this editorial down track.